app/bundles/CampaignBundle/Tests/Command/AbstractCampaignCommand.php

```diff
@@ -128,11 +128,12 @@ protected function createLead(string $leadName): Lead
         return $lead;
     }
 
-    protected function createCampaign(string $campaignName): Campaign
+    protected function createCampaign(string $campaignName, bool $allowRestart = false): Campaign
     {
         $campaign = new Campaign();
         $campaign->setName($campaignName);
         $campaign->setIsPublished(true);
+        $campaign->setAllowRestart($allowRestart);
```

Comment:
I'm not a big fan of adding more and more parameters to the methods. This ups the cognitive load. Because the campaign is an object you can freely change it whenever needed later in the test.

```suggestion
    protected function createCampaign(string $campaignName): Campaign
    {
        $campaign = new Campaign();
        $campaign->setName($campaignName);
        $campaign->setIsPublished(true);
```

Reply:
Sure, I've changed this.